bedrock/mozorg/templates/mozorg/home/home-de.html

```diff
@@ -36,9 +36,9 @@ <h1>{{ self.page_title() }}</h1>
   {% endcall %}
 
   {{ fxa_banner(
-    title=_('Firefox'),
-    sub_title=_('Dein Leben. <br>Deine Daten.'),
-    link_cta=_('Weitere Infos'),
+    logo_title=_('Firefox'),
+    title=_('Dein Leben. <br>Deine Daten.'),
+    link_cta=_('Learn More'),
```

Comment:
Copy-paste missed the Germany CTA. 🍝

Reply:
This is one of the banners. Does German have the other one? My German is a little rusty* so I'm not sure which one this is.

* I never learned any German.

Reply:
Ah whoops. Yes, will update `download_banner` also. 🍝 😅